modules/python/init.zsh

```diff
@@ -65,6 +110,12 @@ if (( $+commands[pip] )); then
   source "$cache_file"
 
   unset cache_file
+
+  # Load auto workon cwd hook
+  if zstyle -t ':prezto:module:python' enable-virtualenv-auto-workon-cwd 'yes'; then
```

Comment:
Hm, what about `zstyle ':prezto:module:python:virtualenv' auto-switch 'yes'`?

It lines up with the ruby module and conveys the same meaning.

Once we decide on a final setting name, it should be added to the sample zpreztorc. Other than that, I think this looks ready.